go/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.8
+
+WORKDIR /go/src/github.com/Originate/exocom/go/
+COPY . .
+
+WORKDIR exocom
+RUN go install
+RUN rm -rf /go/src/
+
+CMD /go/bin/exocom
```

Comment:
What is this Dockerfile for? If for shipping Exocom, why can't we simply copy only the exocom binary into an empty Docker container? Or maybe one based on Busybox or Alpine Linux?